packages/kit/types/internal.d.ts

```diff
@@ -139,12 +155,7 @@ export interface SSRRenderOptions {
 export interface SSRRenderState {
 	fetched?: string;
 	initiator?: SSRPage | null;
-	prerender?: {
-		fallback: string;
-		all: boolean;
-		dependencies: Map<string, ServerResponse>;
-		error: Error;
```

Comment:
Given tests pass it is probably no longer required, but this error type is lost.

Reply:
Yeah, I removed it since it looked unused